benchmark/misc/freelist.js

```diff
@@ -9,7 +9,9 @@ const bench = common.createBenchmark(main, {
 });
 
 function main({ n }) {
-  const { FreeList } = require('internal/freelist');
+  let FreeList = require('internal/freelist');
+  if (FreeList.FreeList)
```

Comment:
Is this needed to benchmark the old version?

Reply:
Correct.

Reply:
Ok, thanks.